cts.json

```diff
@@ -3454,6 +3454,36 @@
         "a𐄁b"
       ]
     },
+    {
+      "name": "functions, match, dot matcher on \\u2028",
```

Comment:
Would it be possible to use \u2028 in the JSON document?

Reply:
Not sure I understand the question. Do you mean removing the second `\`? That would make the test name indistinguishable from the other one.

Reply:
No, I was talking about using `\u2028` in the `"document"` member.

Reply:
It is how it's defined in the document, you can see it in the source file. I'm not sure if it would be possible to keep it as `\uXXXX` in the compiled `cts.json`.